pandas/tests/sparse/test_indexing.py

```diff
@@ -537,6 +542,28 @@ def test_loc_slice(self):
                                   orig.loc['A':'B'].to_sparse())
         tm.assert_sp_series_equal(sparse.loc[:'B'], orig.loc[:'B'].to_sparse())
 
+    def test_reindex(self):
+        orig = self.orig
+        sparse = self.sparse
+
+        res = sparse.reindex([('A', 0), ('C', 1)])
+        exp = orig.reindex([('A', 0), ('C', 1)]).to_sparse()
+        tm.assert_sp_series_equal(res, exp)
+
+        # On specific level:
+        res = sparse.reindex(['A', 'C', 'B'], level=0)
+        exp = orig.reindex(['A', 'C', 'B'], level=0).to_sparse()
+        tm.assert_sp_series_equal(res, exp)
+
+        # single element list (GH 15447)
+        res = sparse.reindex(['A'], level=0)
```

Comment:
Added test for ``copy``, others should be already tested in ``pandas/test/series/missing.py``, e.g. [line 659](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/series/test_missing.py#L659).